Flank/flank
# feat: Auto update dependencies

Fixes #848

## Test Plan
> How do we know the code works?

1. Generate report using command ` ./gradlew dependencyUpdates -DoutputFormatter=json -DoutputDir=..`
2. Build flank scripts using script ` ./flank-scripts/bash/buildFlankScripts.sh`
3. Run `./flank-scripts/bash/flankScripts dependencies update`

`Versions.kt` and `gradle-wrapper.properties` files should be updated

Github action will run each Monday on 5 AM and will create new PR with a dependency update
![image](https://user-images.githubusercontent.com/65554637/93627585-71c84a00-f9e5-11ea-9987-4c423bb4d00c.png)

[Working example](https://github.com/piotradamczyk5/flank/pull/3)
## Checklist

- [x] Update gradle
- [x] Update plugins
- [x] Create scheduled Github Action workflow
